Library/Homebrew/version/null.rb

```diff
@@ -77,12 +77,12 @@ def major_minor_patch
       self
     end
 
-    sig { override.returns(Float) }
+    # sig { override.returns(Float) }
```

Comment:
This is not ideal. If we're losing typing then I wonder whether it's even worth delegating these.